vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/SetVariableSteps.java

```diff
@@ -278,6 +281,39 @@ public void saveTableToContext(Set<VariableScope> scopes, String variableName)
         });
     }
 
+    /**
+     * Gets the value of <b>CSS property</b> from element located by <b>locator</b> and saves it to the <b>variable</b>
+     * with the specified <b>variableName</b>
+     * <p>
+     * Actions performed at this step:
+     * </p>
+     * <ul>
+     * <li>Saves the value of the <i>cssProperty</i> to the <i>variableName</i>
+     * </ul>
+     * @param cssProperty    The name of the CSS property (for ex. 'color', 'flex')
+     * @param locator        The locator to find an element
+     * @param scopes         The set (comma separated list of scopes e.g.: STORY, NEXT_BATCHES) of variable's scope<br>
+     *                       <i>Available scopes:</i>
+     *                       <ul>
+     *                       <li><b>STEP</b> - the variable will be available only within the step,
+     *                       <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     *                       <li><b>STORY</b> - the variable will be available within the whole story,
+     *                       <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     *                       </ul>
+     * @param variableName   The name of the variable to save the CSS property value
+     */
+    @When("I save `$cssProperty` CSS property value of element located by `$locator` to $scopes"
+            + " variable `$variableName`")
+    public void saveCssPropertyValue(String cssProperty, Locator locator, Set<VariableScope> scopes,
+            String variableName)
+    {
+        baseValidations.assertElementExists("The element to get the CSS property value", locator)
+                .map(e -> webElementActions.getCssValue(e, cssProperty))
+                .ifPresentOrElse(value -> saveVariable(scopes, variableName, value),
+                        () -> softAssert.recordFailedAssertion(
+                                String.format("The '%s' CSS property does not exist", cssProperty)));
```

Comment:
it looks like this message will be also logged if element is not found, but it should be logged only if css prop doesn't exist